src/Utilities/XMLWriter.h

```diff
@@ -0,0 +1,38 @@
+////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source
+// License.  See LICENSE file in top directory for details.
+//
+// Copyright (c) 2018 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by:
+// Mark Dewing, mdewing@anl.gov,
+//    Argonne National Laboratory
+//
+// File created by:
+// Mark Dewing, mdewing@anl.gov,
+//    Argonne National Laboratory
+////////////////////////////////////////////////////////////////////////////////
+
+/** @file XMLWriter.h
+ */
+
+#ifndef QMCPLUSPLUS_XML_WRITER_H
+#define QMCPLUSPLUS_XML_WRITER_H
+
+#include <tinyxml/tinyxml2.h>
```

Comment:
shall we put the 'tinyxml' directory under 'Utilities'?

Reply:
That would get it out of the top-level src/ directories.